openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -493,66 +473,137 @@
      bottom: 0;
    }
 
-  #oa_rubric_editor_wrapper{
-
-    .wrapper-comp-settings{
-      display: block;
-    }
-
-    #openassessment_rubric_instructions{
+  .openassessment_tab_instructions {
       background-color: $edx-gray-t1;
       border-bottom: 1px solid $edx-gray-d1;
       padding: 10px;
+  }
+
+  .openassessment_container_header {
+    margin: 10px;
+    padding-bottom: 5px;
+    border-bottom: 1px solid;
+    overflow: auto;
+    color: $edx-gray-d2;
+  }
+
+  .action.expand-collapse {
+    float: left;
+    .ui-toggle-expansion {
+      color: $edx-gray-d2;
+      cursor: pointer;
     }
+    .ui-toggle-expansion:hover {
+      color: $edx-gray-d1;
+      cursor: pointer;
+    }
+  }
 
-    .openassessment_criterion {
+  .openassessment_container_header_title_box {
+    float: left;
+    width: 80%;
+    display: inline-block;
+  }
 
-      .openassessment_criterion_remove_button{
-        @extend .openassessment_rubric_remove_button;
-      }
+  .openassessment_container_header_title {
+    text-transform: uppercase;
+    cursor: default;
+    padding-top: 2px;
+  }
 
-      .openassessment_criterion_header {
-        margin: 10px;
-        padding-bottom: 5px;
-        border-bottom: 1px solid;
-        overflow: auto;
-        color: $edx-gray-d2;
+  .openassessment_container_guide {
+    @extend %t-small;
+  }
 
-        .action.expand-collapse {
-          @include float(left);
-          .ui-toggle-expansion {
-            color: $edx-gray-d2;
-            cursor: pointer;
-          }
-          .ui-toggle-expansion:hover {
-            color: $edx-gray-d1;
-            cursor: pointer;
-          }
-        }
+  .openassessment_container_remove_button{
+    cursor: pointer;
+    h2:after{
+      font-family: FontAwesome;
+      content: "\f00d";
+      display: inline-block;
+      color: inherit;
+      margin: 0 5px;
+    }
+    h2{
+      text-transform: uppercase;
+      font-size: 80%;
+      float: right;
+      display: inline-block;
+      padding: 3px 8px 3px 13px;
+    }
+    float: right;
+  }
+  .openassessment_container_remove_button:hover{
+    background-color: $edx-gray-d2;
+    border-radius: 4px;
+    color: white;
+  }
 
-        .openassessment_criterion_header_title_box {
-            @include float(left);
-            width: 80%;
-            display: inline-block;
-        }
+  .openassessment_container_add_button {
+    h6:before{
+      font-family: "FontAwesome";
+      display: inline-block;
+      margin-left: 5px;
+      margin-right: 10px;
```

Comment:
Do these need to be @include margin-left(5px) and @include margin-right(10px) to support RTL?